README.md

```diff
@@ -97,15 +94,34 @@ crypto-config/peerOrganizations/org1.example.com/users/User1@org1.example.com/ms
 
 ### Run
 
+#### Binary
+
 Execute `./tape config.yaml 40000` to generate 40000 transactions to Fabric.
 
+#### Docker
+
+```
+docker run -v $PWD:/tmp guoger/tape tape $CONFIG_FILE 40000
+```
+
 *Set this to integer times of batchsize, so that last block is not cut due to timeout*. For example, if you have batch size of 500, set this to 500, 1000, 40000, 100000, etc.
 
+## Tips
 
-## Development
+- Put this generator closer to Fabric, or even on the same machine. This is to prevent network bandwidth from being the bottleneck.
```

Comment:
Why not add "You also can use tools like `iftop` to monitor network traffic.  " at the last.  I think we give users some hint is better, don't you think so?

Reply:
https://github.com/guoger/tape/pull/114#discussion_r535768386

i really don't have strong opinion on this.

Reply:
I am fine with current context.
I am not sure, if we'd better refactor this part with link to https://prometheus.io/docs/introduction/overview/ (if we suggestion user use prometheus...)

so that tips can be refactor as:
tips:
1. network samples....
1. monitor tool suggestion....

and I suppose later we are able to use some result from probe to replace `Increase number of messages per block in your channel configuration may help`, as `adjust parameter for xxx may help for sample and ... `

but I suppose the content update can be in further version/pr.